Backend/TodoList.Api/TodoList.Api.UnitTests/TodoList.Api.UnitTests.csproj

```diff
@@ -1,19 +1,19 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>net5.0</TargetFramework>
+    <TargetFramework>net7.0</TargetFramework>
```

Comment:
maybe it's good to leave the version to an outdated one that is still supported by Microsoft (.net 6?), so candidates can showcase extra skills if they update the project to latest version?

Reply:
@michaelm-clearpoint shall I change target framework to .NET 6.

Reply:
yes for me, what you think @mhurbuns @mniem @MikeNguyenCP @gowri-cp ?

Reply:
yes,  its a good way to check if the candidate will upgrade or not

Reply:
I have changed to .NET 6.0

Reply:
LGTM